shacl12-profiling/index.html

```diff
@@ -420,10 +420,10 @@
 	<body>
 		<section id="abstract">
 			<p>
-				This document defines extensions of the SHACL Shapes Constraint Language created to allow for the profiling of specifications.
+				This document defines extensions of the SHACL Shapes Constraint Language created to allow for profiling.
 			</p>
 			<p>
-				SHACL is a language for validating RDF graphs against a set of conditions, so this document's scope is limited to the profiling of RDF graph models, including SHACL-defined models.
+				SHACL is a language for validating RDF graphs against a set of conditions, so this document's scope is limited to profiling of RDF graph models, including SHACL-defined models.
```

Comment:
```suggestion
				SHACL is a language for validating RDF graphs against a set of conditions; this document's scope is limited to profiling RDF graph models, including SHACL-defined models.
```

Reply:
Applied in https://github.com/w3c/data-shapes/pull/489/commits/adb648c749af114ec3c38437e3af9b4c1538cc3c